Homework03

Ukrocoder opened the pull request:
Здесь есть задания с 1 по 6 по порядку.

Ukrocoder committed:
Add files via upload

yushkov-i reviewed (changes requested):
Все работает, отлично.
Тем не менее, старайся использовать переменные для хранения промежуточных результатов расчетов. Например, в задании №4 программисту, который будет изучать / использовать такой  код будет не понятно, что происходит и почему оно работает.
Плюс, как говорилось на занятии переменные упрощают процесс отладки.
Так что в качестве упражнения переделай задание №4 с использованием трех локальных переменных.

Ukrocoder committed:
Add files via upload

Я изменил последнюю задачу, так как нашёл в ней недочёт. Программа могла вывести 1, если цифры левой и правой пары совпадают, даже если они не симметричны.

Ukrocoder committed:
Delete Arithmetic.cpp

Ukrocoder commented:
Нет, там получается деление на ноль, первый вариант лучше.

yushkov-i committed:
Revert "Delete Arithmetic.cpp"

This reverts commit 0faa7d2153274a93f9850e768b4d4a6a0599689a.

yushkov-i committed:
Revert "Add files via upload"

This reverts commit 43a12055f78ee63940712c40806c2777feb58547.

yushkov-i committed:
Rename Arithmetic.cpp to 03/arithmetic.cpp
